src/tests/services/edge-application-functions-services/delete-edge-application-functions-service.test.js

```diff
@@ -11,33 +16,65 @@ const makeSut = () => {
 }
 
 describe('EdgeApplicationFunctionsServices', () => {
-  it('should call Api with correct params', async () => {
+  it('should return the API base url to data streaming service', async () => {
```

Comment:
should return the API base url to data streaming service => como o teste não retorna nada nem é feito expect sobre retorno após chamar a API, sugiro um título diferente:
should return the API base url to data streaming service -> should call API with correct params. Pois estamos no expect verificando a url e o Method usados.